web/client/components/geostory/contents/Text.jsx

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright 2019, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import React from 'react';
+
+
+const Text = (props) => {
+    return (
+        <div className="geostory-content-text" style={{margin: 20}} onClick={() => {
```

Comment:
We could use a className like 'ms-geostory-content--text' or similar and remove margin 20 from here